CMakeLists.txt

```
@@ -78,6 +78,9 @@ OPTION(WANT_VST_32	"Include 32-bit VST support" ON)
 OPTION(WANT_VST_64	"Include 64-bit VST support" ON)
 OPTION(WANT_WINMM	"Include WinMM MIDI support" OFF)
 OPTION(WANT_DEBUG_FPE	"Debug floating point exceptions" OFF)
+option(WANT_ASAN	"Enable AddressSanitizer" OFF)
+option(WANT_TSAN	"Enable ThreadSanitizer" OFF)
+option(WANT_MSAN	"Enable MemorySanitizer" OFF)
```

Let's specify that these are technically debugging options?
```suggestion
option(WANT_DEBUG_ASAN	"Enable AddressSanitizer" OFF)
option(WANT_DEBUG_TSAN	"Enable ThreadSanitizer" OFF)
option(WANT_DEBUG_MSAN	"Enable MemorySanitizer" OFF)
```